python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -746,8 +832,16 @@ def add_edge_data(
 
     def get_edge_data(self, edge_ids=None, types=None, columns=None):
         """
-        Return a dataframe containing edge properties for only the specified
+        Create a dataframe containing edge properties for only the specified
         edge_ids, columns, and/or edge type, or all edge IDs if not specified.
+
+        Parameters
+        ----------
+        edge_ids :
+        types : list of edge types to include in returned dataframe, optional
+            None is the default and will allow any edge type
```

Comment:
It looks like the format here is different than in the descriptions above.  For example, shouldn't this look more like:
```suggestion
        types : list, optional
            List of edge types to include in returned dataframe. None is the default and will allow any edge type.
```

Reply:
Fixed